estop/code/SPI_rf24L01_RX/SPI_rf24L01_RX.ino

```diff
@@ -0,0 +1,219 @@
+          digitalWrite(BTN_LED, LOW);
+        }
+    }
+    Serial.println(" ");
+  }
+  SPI_RW_Reg(WRITE_REG+STATUS,status);                             // clear RX_DR or TX_DS or MAX_RT interrupt flag
+  delay(25);
+}
+
+//**************************************************
+// Function: init_io();
+// Description:
+// flash led one time,chip enable(ready to TX or RX Mode),
+// Spi disable,Spi clock line init high
+//**************************************************
+void init_io(void)
+{
+  digitalWrite(IRQ, 0);
+  digitalWrite(CE, 0);			// chip enable
+  digitalWrite(CSN, 1);                 // Spi disable	
+}
+
+/************************************************************************
+**   * Function: SPI_RW();
+ * 
+ * Description:
+ * Writes one unsigned char to nRF24L01, and return the unsigned char read
+ * from nRF24L01 during write, according to SPI protocol
+************************************************************************/
+unsigned char SPI_RW(unsigned char Byte)
+{
+  return SPI.transfer(Byte);
+}
+
+/**************************************************/
+
+/**************************************************
+ * Function: SPI_RW_Reg();
+ * 
+ * Description:
+ * Writes value 'value' to register 'reg'
+/**************************************************/
+unsigned char SPI_RW_Reg(unsigned char reg, unsigned char value)
+{
+  unsigned char status;
+
+  digitalWrite(CSN, 0);                   // CSN low, init SPI transaction
+  SPI_RW(reg);                            // select register
+  SPI_RW(value);                          // ..and write value to it..
+  digitalWrite(CSN, 1);                   // CSN high again
+
+  return(status);                   // return nRF24L01 status unsigned char
+}
+/**************************************************/
+
+/**************************************************
+ * Function: SPI_Read();
+ * 
+ * Description:
+ * Read one unsigned char from nRF24L01 register, 'reg'
+/**************************************************/
+unsigned char SPI_Read(unsigned char reg)
+{
+  unsigned char reg_val;
+
+  digitalWrite(CSN, 0);                // CSN low, initialize SPI communication...
+  SPI_RW(reg);                         // Select register to read from..
+  reg_val = SPI_RW(0);                 // ..then read register value
+  digitalWrite(CSN, 1);                // CSN high, terminate SPI communication
+
+  return(reg_val);                     // return register value
+}
+/**************************************************/
+
+/**************************************************
+ * Function: SPI_Read_Buf();
+ * 
+ * Description:
+ * Reads 'unsigned chars' #of unsigned chars from register 'reg'
+ * Typically used to read RX payload, Rx/Tx address
+/**************************************************/
+unsigned char SPI_Read_Buf(unsigned char reg, unsigned char *pBuf, unsigned char bytes)
+{
+  unsigned char sstatus,i;
+
+  digitalWrite(CSN, 0);                   // Set CSN low, init SPI tranaction
```

Comment:
While this work, since you are writing all of these explicitly and not in a variable you might want to use HIGH and LOW for clarity